crates/exex/exex/src/wal/mod.rs

```diff
@@ -25,7 +25,7 @@ use reth_tracing::tracing::{debug, instrument};
 ///    with [`Wal::commit`].
 /// 3. When the chain is finalized, call [`Wal::finalize`] to prevent the infinite growth of the
 ///    WAL.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
if we intend to share this type, then we could make this slightly nicer by only using one Arc here with an Inner type, but since we don't expect a ton of clones, this is fine